tasks/kickstart-guest.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Create a new guest using Kickstart
+
+- name: copy over kickstart file
+  template: src="kickstart.cfg" dest="{{ kickstart_tempdir }}/{{ inventory_hostname }}.ks"
```

Comment:
Ok, happy to fix this one. I actually kept it so you wouldn't need to modify your other code!